fastlane/Fastfile

```diff
@@ -61,6 +61,58 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases and opens a PR"
+lane :open_pr_upgrading_dependencies do |options|
+  if options[:dry_run]
+    dry_run = true
+  end
+  
+  new_versions = update_native_dependencies_to_latest()
+  
+  repo_status = Actions.sh("git status --porcelain")
+
+  repo_clean = repo_status.empty?
+
+  if repo_clean
+    UI.success('Git status is clean, nothing to update! 💪')
+    next
+  end
+
+  new_ios_version = new_versions[:new_ios_version]
+  new_android_version = new_versions[:new_android_version]
+
+  branch = "update-#{new_ios_version}-#{new_android_version}"
+  
+  result = Actions.sh("git checkout -b #{branch}")
```

Comment:
Hmm I would think creating a new branch shouldn't be done in a `dry-run`, same with committing changes. Maybe not do those in a `dry-run` and just print the versions?

Reply:
Also, this will checkout from the current branch.... I wonder if we should ensure this is executed from the `main` branch and that it's updated... I guess we can add those validations later if needed.

Reply:
I am going to change the command to be `git checkout -b #{branch} main` so that the branch is off main